src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $day-block-size * 7 + $gap-size * 6 + $side-padding * 2;
+  padding: 25px $side-padding $side-padding;
+  gap: $gap-size;
+
+  &__day {
+    box-sizing: border-box;
+    width: $day-block-size;
+    height: $day-block-size;
+    border: 1px solid #000;
+
+    background-color: #eee;
+    text-align: center;
+    transition: 0.5s;
```

Comment:
Would be great to make transition-duration a variable as it should be the same for whole page.
Also, it's better to specify transition-properties
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/e429a5f1-a1bb-4ad0-b12e-f50d6d6caee7)